espd-web/src/main/webapp/WEB-INF/views/criteriaTemplates/formTemplate.jsp

```diff
@@ -65,7 +65,7 @@
 				<div class="col-md-12">
 					<div class="form-group">
 						${span18n[allows_checks?"crit_do_you_allow_checks":"crit_your_answer"]}
-						<c:set var="checktarget" value="${'#'.concat(field).concat('-form')}"/>
+						<c:set var="checktarget" value="#${field}-form"/>

```

Comment:
Because of the JSP compilation, the $ symbol is not resolved as a beginning of an EL expression because of the '#' character.
It is even better to disable the Eclipse validators if you want to have faster build times and they are not always correct and reliable.